src/utils/request.js

```diff
@@ -51,4 +51,26 @@ const request = extend({
   errorHandler, // extend default error handler with custom actions
 });
 
+request.interceptors.request.use((url, options) => {
+  const token = localStorage.getItem('token');
+  if (token) {
+    try {
+      const tokenHeader = {
+        token,
```

Comment:
shouldn't this be `'Authorization: Bearer ' + token`? Or is there some additional magic happening behind the scenes?

Reply:
Yes Dave, this is just for testing purposes, ideally this will be a Bearer + tokenValue

Reply:
Fixed now!